budgetportal/summaries.py

```diff
@@ -553,3 +557,55 @@ def get_aggregate_drilldowns(self):
             openspending_api.get_programme_name_ref(),
             openspending_api.get_econ_class_4_ref(),
         ]
+
+
+class BudgetedAndActualComparison(DepartmentBudgetData):
```

Comment:
```suggestion
class InYearSpending(DepartmentBudgetData):
```

This is not the Budgeted vs actual dataset, it's the In Year Spending dataset.

Reply:
done